marklogic-data-hub/build.gradle

```diff
@@ -247,46 +247,70 @@ ext {
     }
 }
 
-//task createLocalFile() {
-//	boolean sslRun = Boolean.parseBoolean(System.properties['ssl'])
-//	boolean certAuth = Boolean.parseBoolean(System.properties['certauth'])
-//	com.marklogic.mgmt.ManageClient mC = new ManageClient(new com.marklogic.mgmt.ManageConfig());
-//	com.marklogic.mgmt.resource.hosts.HostManager hm = new HostManager(mC);
-//	def	bootStrapHost = hm.getHostNames().get(0);
-//	def fileName = "gradle-local.properties"
-//	def inputFile = new File(fileName)
-//	if(sslRun) {
-//		inputFile.write("# Put your overrides from gradle.properties here\n" +
-//    			"# Don't check this in to version control\n" +
-//    			"mlJobSimpleSsl=true\n" +
-//    			"mlTraceSimpleSsl=true\n" +
-//    			"mlFinalSimpleSsl=true\n" +
-//    			"mlAdminScheme=https\n" +
-//		        "mlManageScheme=https\n" +
-//    			"mlStagingSimpleSsl=true")
-//	}
-//
-//	if(certAuth) {
-//		inputFile.write("# Put your overrides from gradle.properties here\n" +
-//				"# Don't check this in to version control\n" +
-//				"mlStagingAuth=certificate\n" +
-//	    		"mlTraceAuth=certificate\n" +
-//	    		"mlFinalAuth=certificate\n" +
-//	    		"mlHost="+bootStrapHost+"\n"+
-//	    		"mlAdminScheme=https\n" +
-//	   	        "mlManageScheme=https\n" +
-//	    		"mlJobAuth=certificate")
-//	}
-//}
+def propFile = file("gradle.properties")
+String fileContents = propFile.text
+boolean sslRun = Boolean.parseBoolean(System.properties['ssl'])
+boolean certAuth = Boolean.parseBoolean(System.properties['certauth'])
+def props = new Properties()
+propFile.withInputStream { props.load(it) }
```

Comment:
All these properties should just be available to build.gradle without having to read the file.
the variable names should just be `certauth`, `ssl` etc.  Maybe you were having trouble reading them that way?

Reply:
1. The values for 'certauth' and 'ssl' are provided in command line . For example, ./gradlew marklogic-data-hub:test -Dcertauth=true (or -Dssl=true). So, I am reading them from System.properties
2. I think I can use other properties like 'mlHost', 'mlSecurityUsername' and 'mlSecurityPassword' directly. I will make that change.